app.rb

```diff
@@ -0,0 +1,45 @@
+require './book'
+require './person'
+require './teacher'
+require './student'
+
+class App
+  def list_all_books(books, show_list: false)
+    puts 'No list of books are found' if books.empty?
+    books.each_with_index do |book, list|
+      puts "#{"#{list})" if show_list} Title: #{book.title}, Author: #{book.author}"
+    end
+    puts "\n"
+  end
+
+  def list_all_people(people, show_list: false)
+    puts 'No list of people are found' if people.empty?
+    people.each_with_index do |person, list|
+      puts "#{"#{list})" if show_list} [#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+    puts "\n"
+  end
+
+  def create_person(select, age, name, specialization, parent_permission)
+    case select
+    when 1
+      Student.new(age, name, parent_permission)
+    when 2
+      Teacher.new(age, name, specialization, parent_permission: parent_permission)
+    end
+  end
+
+  def list_rentals_for_person_id(persons, id)
+    return persons.find { |person| person.id == id }.rentals if persons.any? { |person| person.id == id }
+
+    puts 'No rentals found for this ID'
+  end
```

Comment:
- [ ] After successfully adding a rental to the app, I couldn't see or list it. Kindly fix this issue for the app to meet its requirements.

![rental](https://user-images.githubusercontent.com/53124548/231983822-b2b6b32b-e77c-440f-9e9c-d54b1363c39f.png)